src/Microsoft.AspNet.OData.Shared/DefaultContainerBuilder.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.AspNet.OData
     /// <summary>
     /// The default container builder implementation based on the Microsoft dependency injection framework.
     /// </summary>
-    public class DefaultContainerBuilder : IContainerBuilder
+    public class DefaultContainerBuilder : Microsoft.OData.IContainerBuilder
```

Comment:
>Microsoft.OData. [](start = 43, length = 16)

did you opt for this approach rather than adding a using statement because it was easier to find replace with this approach? bit of a nit, i guess, but if this were just a single file change we'd prefer to see the using statement.

Reply:
I thought it was because of confusion between OData vs DependencyInjection classes but that appears to not be the case, the using works fine in this class.